mantidimaging/gui/widgets/mi_image_view/view.py

```diff
@@ -211,12 +212,12 @@ def _update_roi_region_avg(self) -> SensibleROI | None:
 
         if self.roi.isVisible():
             z_value = int(self.timeLine.value())
-            mean_val = self.image[z_value, top:bottom, left:right].mean()
+            mean_val = self.image[z_value, top:bottom, left:right].mean() if self.image is not None else 0
```

Comment:
Interesting that it warns about this, but not line 206. Would it is work to put a `assert self.image is not None` at the top of the method? or can this be called without the image being set?